src/SignhostAPIClient/Rest/ISignhostApiClientSettings.cs

```diff
@@ -18,6 +18,8 @@ public interface ISignHostApiClientSettings
 		/// </summary>
 		string APPKey { get; }
 
+		string SharedSecret { get; }
```

Comment:
Unfortunately this breaks backward compatibility ... unfortunately we don't have default interface methods (yet), not sure how to fix this without bumping the package